javascript/modules/client/test/unit/client.test.ts

```diff
@@ -182,7 +182,10 @@ describe("Client instances", () => {
 
     const gasFeesEstimation = await client.estimate.create(daoCreationParams);
 
-    expect(BigNumber.isBigNumber(gasFeesEstimation)).toBeTruthy();
+    expect(typeof gasFeesEstimation).toEqual("object");
+    expect(BigNumber.isBigNumber(gasFeesEstimation.avg)).toBeTruthy();
+    expect(BigNumber.isBigNumber(gasFeesEstimation.max)).toBeTruthy();
+    expect(gasFeesEstimation.max.gt(gasFeesEstimation.avg)).toBeTruthy();
```

Comment:
I would definitely add a test to check that the average value is lower than the max one. This should surface that issue I commented above

Reply:
It's already done in line 188